compiler/crates/relay-compiler/src/build_project/artifact_generated_types.rs

```diff
@@ -27,8 +30,9 @@ impl ArtifactGeneratedTypes {
         operation: &OperationDefinition,
         skip_types: bool,
         is_client_only: bool,
+        project_config: &ProjectConfig,
     ) -> Self {
-        if skip_types {
+        if skip_types || project_config.typegen_config.language == TypegenLanguage::TypeScript {
```

Comment:
Not related to your change here, but I'm kinda bummed how this works. Having the set of imports defined far away from the code that uses them is pretty brittle. Ideally we could have a setup where the code that references an imported type also adds that type to a set, and then when we go to print the imports we use a sorted version of that set.

Does not need to be in scope for this PR, but would be a nice improvement for someone to do at some point.